library/psa_crypto.c

```diff
@@ -1088,13 +1088,10 @@ static psa_status_t psa_get_and_lock_transparent_key_slot_with_policy(
 
 psa_status_t psa_remove_key_data_from_memory(psa_key_slot_t *slot)
 {
-    /* Data pointer will always be either a valid pointer or NULL in an
-     * initialized slot, so we can just free it. */
```

Comment:
I removed this stale comment, which clearly refers to the use of `mbedtls_free()`, where freeing a `NULL` pointer is defined as a no-op